code/mobile/android/PhoneVR/app/src/noGvr/res/drawable/rippleable.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Background that displays a white transparent color when touched.
+  This does no actual rippling pre-21, but is overridden on 21 to use a RippleDrawable.
```

Comment:
Does this fix something or just some UI design ?

Reply:
required for successful build, these were previously implicitly copied from GVR SDK